src/rustup-cli/download_tracker.rs

```diff
@@ -121,37 +121,31 @@ impl DownloadTracker {
     }
     /// Display the tracked download information to the terminal.
     fn display(&mut self) {
-        let total_h = HumanReadable(self.total_downloaded as u64);
+        let total_h = HumanReadable(self.total_downloaded as f64);
         let sum = self.downloaded_last_few_secs
                       .iter()
-                      .fold(0usize, |a, &v| a + v);
+                      .fold(0., |a, &v| a + v as f64);
```

Comment:
Hmm, this is just a matter of taste. I just wanted `sum` to be a float and I tend to apply int-to-float casting as early as possible when I know the end result is a float.